src/oscar/test/factories/order.py

```diff
@@ -59,7 +60,9 @@ class Meta:
     if hasattr(settings, 'OSCAR_INITIAL_ORDER_STATUS'):
         status = settings.OSCAR_INITIAL_ORDER_STATUS
 
-    site_id = settings.SITE_ID
+    site_id = factory.LazyAttribute(
```

Comment:
Would everything work properly if you just do:
```
site_id = getattr(settings, "SITE_ID", None)
```
That would be preferable to me because it is much simpler. Would the tests still pass that way?

Reply:
Yes indeed, that would work as well :)